### ISSUE TITLE
bug: screen readers announce the icon name for ion-back-button when they should not

### ISSUE THREAD

**Issue by terrancle** [ionic-team/ionic#21043]
Please add aria-hidden="true" on ionIcon in ionBackButton.
Actualy, VoiceOver display the icon name in aria-label.
Please add aria-hidden on ionIcon, or change value of aria-label with the text atribut value.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you please describe the issue you are running into?

**Comment by terrancle** [comment]
If I define the ionBackButton tag with the icon = "chevron-back-outline" and text = "Back" parameters, VoiceOver says "Chevron back outline back" when it navigates to the back button of the page.
In the generated html code, we can see that an aria-label attribute "is placed via the ionIcon tag, with the value of icon =" chevron-back-outline ". The aria-label attribute should contain the value of text = "XXX" and not icon = "XXX" or that the back button icon contains an aria-hidden = "true".
With this last option, VoiceOver will only see the value of the text = "" attribute.

**Comment by liamdebeasi** [comment]
Thanks for the follow up. I tested this out, and VoiceOver reads the icon name, the value of the text, and then indicates that it is a button. I am not able to reproduce the scenario where the text value is not read. The `aria-label` is used to describe a label that is not visible on the screen. In this case, we use `aria-label` to describe the `ion-icon` as that is not text. Since the button text is visible, the screen reader should pick that up automatically.

Is the issue you are describing that the icon name is being read in addition to the button text, or is the issue that the button text is not being read at all?

**Comment by terrancle** [comment]
The name of the icon is read, that's precisely the problem.
You just need to put an aria-hidden on the icon, so that VoiceOver only reads the text.

**Comment by liamdebeasi** [comment]
Thanks for the follow up. Can you try the following dev build and let me know if it resolves the issue?

Ionic Angular
`npm i @ionic/angular@5.1.0-dev.202004152241.fe17538`

Ionic React
`npm i @ionic/react@5.1.0-dev.202004152241.fe17538`

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved via https://github.com/ionic-team/ionic/pull/21053, and a fix will be available in an upcoming release of Ionic Framework.
